components/Banner/tests/unit/Theming/MDCBannerViewThemingTests.m

```diff
@@ -0,0 +1,125 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <XCTest/XCTest.h>
+
+#import "MaterialBanner+Theming.h"
+#import "MaterialBanner.h"
+#import "MaterialContainerScheme.h"
+
+static CGFloat const kBannerViewTextViewOpacityDefault = (CGFloat)0.87;
+static CGFloat const kBannerViewDividerOpacityDefault = (CGFloat)0.12;
+
+@interface MDCBannerViewThemingTests : XCTestCase
```

Comment:
Nit: Please provide header docs for all non-trivial interfaces.  Includes this class and probably the properties below.

Reply:
Addressed in fa7f9b4bcf50023911f2ef5534cbcae64791c89c.